src/js/graphql_endpoint/modules/jwt.ts

```diff
@@ -18,7 +18,6 @@ const secretsmanager = new AWS.SecretsManager({
 });
 
 async function fetchJwtSecret(): Promise<string> {
-    console.log("JWT_SECRET_ID: ", JWT_SECRET_ID);
```

Comment:
seems semi suspicious, so, deleted

Reply:
The ID isn't a secret, but yeah it's fine to just remove this.